apitest/styles.css

```diff
@@ -25,6 +25,7 @@ body {
 #controls {
     display: flex;
     flex-direction: column;
+    padding: 2px;
```

Comment:
The apitest controls shows the click count. Currently, when this reaches double-digits, the controls container grows, causing the layout container to change size.  So changes in this file were just to avoid this (because having things change size when you click makes debugging size issues harder :-) )